src/pages/_app.tsx

```diff
@@ -136,7 +136,13 @@ user's mobile device or desktop. See https://developers.google.com/web/fundament
   );
 }
 
-const App = ({ Component, pageProps }) => (
+const App = ({
+  Component,
+  pageProps,
+}: {
+  Component: React.FC;
+  pageProps: any;
+}) => (
```

Comment:
Next.js has a type for this:

```suggestion
const App = ({
  Component,
  pageProps,
}: AppProps) => (
```

Imported using `import type { AppProps } from 'next/app'`